scripts/CodeTestsMultiple.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+#At the moment this is for sb-pim only
+# APP_TYPE=$1 TODO, replace typing.xml with this 
+# TODO : remove hard coded container "pim_api" 
+
+echo "App health check"  # Check to see if the app container is running or not
+sleep 5
+docker-compose exec -T pim_api python -c "import requests; requests.get('http://localhost:8000/health')" || exit 1
+
+echo "Clean up old reports" 
+rm -f unittesting.xml coverage.xml typing-server.xml typing-integrations.xml
+
+echo "Code tests" 
+## Catch the exit codes so we don't exit the whole script before we are done.
+## Typing, linting, formatting check & unit and integration testing
+docker-compose ps
+docker-compose exec -T pim_api flake8; STATUS1=$?
```

Comment:
Wait why don't use use `validatecodeonce` here and do only `flake8`?

Reply:
This was a special case for [sb-pim](https://github.com/SatelCreative/sb-pim/blob/6cf9b95c5fd82796d19b0bd7cc961d4de8ed3275/Jenkinsfile#L108), I just took what was done in that repo

Reply:
That's the lack of flexibility with this scheme that is not ideal ... flake8 only is fine. But Frank should fix the rest at some point.

Reply:
I have reported this to SB-PIM management team and ask them to put in the backlog as they don't have budget for it atm. Frank said it'll be 2 days of work to fix them